### PR TITLE
Add a back button to tag overview (Fixed #2151)

### LINKED ISSUE #2151: missing button to return from tag overview
- go to the menu for motions
- click there on the button "Tags"

result: one sees the list of existing tags. What is missing is a button to go back to motions. For all other things there is such a back button.

### PR DESCRIPTION
**Pull request by matakuka**

### CODE REVIEW

File: openslides/core/static/templates/core/tag-list.html [hunk 1]
```diff
@@ -5,6 +5,10 @@
         <i class="fa fa-plus fa-lg"></i>
         <translate>New</translate>
       </a>
+       <a href="javascript:history.back()" class="btn btn-sm btn-default">
```
Comment (normanjaeckel) [inline openslides/core/static/templates/core/tag-list.html]: Remove one whitespace.

**Comment by normanjaeckel** [comment]
One optional remark. If you do not have time to change this and rebase the git commits, please let me know. I can merge this pull request and we remove the whitespace another day (of life :-) ).

**Comment by emanuelschuetze** [comment]
Ok for me.

**Comment by matakuka** [comment]
@normanjaeckel it's okay. I'm still (re-)learning github and have to squish all my errors, so that I won't repeat them

**Comment by normanjaeckel** [comment]
I am sorry because I merged another pull request before this. Now you have to rebase again on top of current master (because of merge conflicts in the AUTHORS file.). Thanks in advance.

**Comment by matakuka** [comment]
Another thing learned on github :)

File: AUTHORS [hunk 2]
```diff
@@ -20,3 +20,5 @@ Authors of OpenSlides in chronological order of first contribution:
     John Felipe Urrego Mejia <ingenierofelipeurrego@gmail.com> (Spanish translation)
     Erik Steenman <eriksteenman@gmail.com>
     Sean Engelhardt <sean.f.t.engelhardt@gmail.com>
+    Maximilian Krambach <maximilian.krambach@gmx.de>
+
```
Comment (emanuelschuetze) [inline AUTHORS]: Please remove the blank line.

**Pull request merged**